7_4_Movie.py

```diff
@@ -0,0 +1,109 @@
+import random
+from datetime import datetime
+list = []
+movie_list = []
+serial_list = []
+today = datetime.today()
+data1 = today.strftime("%d/%m/%Y")
+
+class MovieCatalog:
+    def __init__(self, title, year, genre, number_review):
+        self.title = title
+        self.year = year
+        self.genre = genre
+        self.number_review = number_review
+
+    def __str__(self):
+        return f"{self.title} {self.year}"
+
+    def play(self, review_step=1):
+        self.number_review += review_step
+
+
+class SerialCatalog(MovieCatalog):
+    def __init__(self, episode_number, sezon_number, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.episode_number = str(episode_number).zfill(2)
+        self.sezon_number = str(sezon_number).zfill(2)
+
+    def __str__(self):
+        return super().__str__() + f" {self.episode_number} {self.sezon_number}"
+
+def Add_Movie(movie_name, movie_year, movie_genre):
+    list.append(movie_name)
+    movie_list.append(movie_name)
+    movie_name = MovieCatalog(movie_name, movie_year, movie_genre, 0)
+
+def add_serial(serial_name, serial_year, serial_genre, serial_episode, serial_sezon):
+    list.append(serial_name)
+    serial_list.append(serial_name)
+    serial_name = SerialCatalog(
+        title=serial_name,
+        year=serial_year,
+        genre=serial_genre,
+        number_review="0",
+        episode_number=serial_episode,
+        sezon_number=serial_sezon
+    )
+
+#losowe dodawanie odtworzeń
+def generate_views():
+    list_len=len(list)
+    movie_add_review = list[random.randrange(list_len)]
+    movie_add_review.play(random.randrange(2, 101))
+
+def xgenerate_views():
+    for i in range (10):
```

Comment:
To jest tak krańcowy przypadek, że nie warto na to robić funkcji
Można też dodać argument count:int do generate_views i użyć random.choices(k=count)